PrimaryInstallScript.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [ $EUID -ne 0 ]; then
+	echo "Please run as root user"
+	exit 1
+fi
+
+ping -q -c1 google.com > /dev/null
+ 
+if [ $? -ne 0 ]; then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#install Epsilon-Hermes
+git clone https://github.com/UCSolarCarTeam/Epsilon-Hermes.git /opt/
+(cd /opt/Epsilon-Hermes && cd ../ && mv ./Epsilon-Hermes ./src && mkdir Epsilon-Hermes && mv ./src ./Epsilon-Hermes/)
+qmake /opt/Epsilon-Hermes/
+make /opt/Epsilon-Hermes/
+/opt/build/SchulichEpsilonHermes
```

Comment:
what does this line do?

Reply:
Upon running the make file, the executable to be run is typically stored in the build directory outside the repo, this line runs the executable to launch the program, perhaps I should remove it here (and below) so that all applications don't run at once upon running the script.